benchmark/fio_benchmark_cleanup.yml

```diff
@@ -38,29 +38,19 @@
     - name: Delete image
       command: "rbd remove {{ fiobench_pool_name | default('fiobench') }}/{{ item }} -k /etc/ceph/ceph.client.fiobench.keyring -n client.fiobench"
       with_items:
-        - "bench_file"
-        - "bench_direct"
+        - "bench_file-{{ inventory_hostname_short }}"
+        - "bench_direct-{{ inventory_hostname_short }}"
 
 # Cleanup the benchmark pool if we created it
 - hosts: mons[0]
   become: True
   tasks:
-    - name: Check the value of mon_allow_pool_delete
```

Comment:
You need these tasks, in Luminous you have to set the mon_allow_pool_delete flag, otherwise it fails (which is why the job is failing).
The other tasks ensure the state remains the same before and after cleanup.

Otherwise I think this looks fine.

Reply:
For completeness you may want to adjust the grep on lines 6 and 10 to match the new name.

Reply:
Hmm.  Yea I moved this into the vars file for the phobos performance lab because the setting was not getting applied without restarting the monitors.  But clearly it is working for the AIO test and by removing it here and not putting it in a vars file for AIO it is no longer getting set.  I'll add it back in and take another look at the phobos performance tests.